CHANGELOG.md

```diff
@@ -32,9 +32,15 @@ Thank you to all who have contributed!
 - Adds `org.partiql.value` (experimental) package for reading/writing PartiQL
   values
 - Adds function overloading to the `CompilerPipeline` and experimental `PartiQLCompilerPipeline`.
+- Adds `EvaluationSessionConnector` class to convert `EvaluationSession` to `ConnectorSession`.
+  - Adds `getFunctions()` to the `org.partiql.spi.Plugin` and `org.partiql.plugins.mockdb.localPlugin`.
+- Adds visitor transformations for `Float32Value` and `Float64Value` in `PartiQLValueVisitor`.
+- **Breaking**: Adds `ServiceLoaderUtil` class to load pluggable functions into CLI.
+- **Breaking**: Adds `PartiQLFunction` interface.
 
 ### Changed
 - Standardizes `org/partiql/cli/functions/QueryDDB` and other built-in functions by the new `ExprFunction` format.
+- Changes the parent class of `Float32Value` and `Float64Value` from `ScalarValue<>` to `NumericValue<>` in `PartiQLValue`.
```

Comment:
As these changes are unreleased, we don't need to call this out in the CHANGELOG.